qcodes/data/hdf5_format.py

```diff
@@ -531,7 +531,7 @@ def write_metadata(self, data_set: 'DataSet', io_manager=None, location=None, re
 
         fn = io_manager.join(location, self.metadata_file)
         with io_manager.open(fn, 'w', encoding='utf8') as snap_file:
-            json.dump(data_set.metadata, snap_file, sort_keys=True,
+            json.dump(data_set.metadata, snap_file, sort_keys=sort_keys,
```

Comment:
sorry didn't notice before - please describe this argument in the docstring of the method.